### PR TITLE
Minor fix to javadoc of the RuleChain.around method (nextRule should …

### PR DESCRIPTION
**Pull request by cmebarrow**
…be enclosedRule, assuming it's referring to the method parameter).

### CODE REVIEW

File: src/main/java/org/junit/rules/RuleChain.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ private RuleChain(List<TestRule> rules) {
     }
 
     /**
-     * Create a new {@code RuleChain}, which encloses the {@code nextRule} with
+     * Create a new {@code RuleChain}, which encloses the {@code enclosedRule} with
```
Comment (kcooney) [inline src/main/java/org/junit/rules/RuleChain.java]: I slightly prefer "...encloses the given {@link Rule} with..." (Javadoc should read like sentences).

Comment (cmebarrow) [inline src/main/java/org/junit/rules/RuleChain.java]: Agreed, will do

**Comment by cmebarrow** [comment]
@kcooney I have reacted to your feedback

**Comment by kcooney** [comment]
@cmebarrow sorry, I've been busy with non-JUnit things.

This looks good. Will merge it soon

**Comment by stefanbirkner** [comment]
Change merged: 1d63100e6d69f448a3afb4571e9915cb9439fe1d

Thanks for your contribution.

**Pull request closed without merging**